### PR TITLE
Feat/1380/modify features for adp projects

### PR DESCRIPTION
**Pull request by GDamyanov**
Feat for [#1380](https://github.com/SAP/open-ux-tools/issues/1380)

The features for all adaptation projects which are loaded from "WorkspaceConnector" in "preview-middleware-client" are with "isVariantAdaptationEnabled=true".

### CODE REVIEW

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]
Please add a `changeset` by executing `pnpm changeset`. The changeset needs to update both `preview-middleware-client` and `preview-middleware` packages.

File: packages/preview-middleware-client/src/flp/WorkspaceConnector.ts [hunk 1]
```diff
@@ -79,6 +80,10 @@ const connector = merge({}, ObjectStorageConnector, {
             features.isVariantAdaptationEnabled = false;
         }
 
+        if(settings?.scenario === 'ADAPTATION_PROJECT') {
```
Comment (zdravko-georgiev) [inline packages/preview-middleware-client/src/flp/WorkspaceConnector.ts:83]: Please add also a space between the `if` and the `(` in order to keep the formatting.

Comment (GDamyanov) [reply packages/preview-middleware-client/src/flp/WorkspaceConnector.ts]: Done

File: .changeset/plenty-forks-act.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': minor
+'@sap-ux/preview-middleware': minor
```
Comment (zdravko-georgiev) [inline .changeset/plenty-forks-act.md:3]: Please change from `minor` to `patch`. This is a small change, we do not need a minor version update

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: .changeset/plenty-forks-act.md (hunk 2 above)
Comment (GDamyanov) [reply .changeset/plenty-forks-act.md]: Done

**Comment by zdravko-georgiev** [comment]
> [CLA assistant check](https://cla-assistant.io/SAP/open-ux-tools?pullRequest=1383) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/SAP/open-ux-tools?pullRequest=1383) before we can accept your contribution.**0** out of **2** committers have signed the CLA.❌ GDamyanov❌ GDamyanovSAPYou have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP/open-ux-tools?pullRequest=1383) it.

@GDamyanov Could you please sign the CLA and also merge the latest `main` branch.

**Review by zdravko-georgiev — APPROVED** [review]
Change is easy to understand and adheres to coding guidelines
Excellent test coverage
Did NOT tested manually

**Review by nikmace — APPROVED** [review]
Changes make sense.
- tested manually, all good
- test coverage is good

**Pull request merged**